.github/workflows/main.yml

```diff
@@ -15,12 +15,10 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [2.7, 3.5, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
@sercant Why are we deprecating support for older Python versions if these code changes are not changing the requirements?

Reply:
because we don't use those python versions and the github CI image won't work because they are not supported in the new linux versions.

either I update this, or the CI image.